documentation/movies.workspace.adoc

```diff
@@ -702,6 +702,20 @@ MATCH path = (person)-[:ACTED_IN]->(m)<-[:DIRECTED]-(d)
 RETURN path;
 ----
 
+To aid query performance, you can add indexes to your data.
+(The dataset you create in this guide is very small, but it is good practice to use indexes as they can greatly improve query performance when the datasets are larger.)
+
+You already have two indexes because of the constaints created in the beginning of the `MERGE` statement, see link:https://neo4j.com/docs/cypher-manual/current/constraints/[Cypher Manual -> Constraints^] for more information.
+
+You may be interested in both the year people were born and the year movies were released.
+Run the following to add indexes on these properties:
+
+[source,cypher]
+----
+CREATE INDEX person_born FOR (p:Person) ON (p.born);
+CREATE INDEX movie_released FOR (m:Movie) ON (m.released)
+----
+
```

Comment:
If you wanted to demonstrate that the creation has happened you could also add something like...

```
You can view the indexes within the database by running the following Cypher statement:

[source, cypher]
SHOW INDEXES
```

This would help tie together the action of running the Cypher with the creation of the indexes and demonstrate that the constraint also created the indexes.